client/web/compose/src/components/Common/Grid.vue

```diff
@@ -142,15 +140,18 @@ export default {
   },
 
   methods: {
-    onBlockUpdated (index) {
-      this.$emit('item-updated', index)
-
-      const { x, y, w, h } = this.layout[index]
-      this.blocks[index].xywh = [x, y, w, h]
-    },
-
     onLayoutUpdated () {
       this.resizing = false
+
+      this.blocks.forEach(({ xywh = [] }, idx) => {
+        const { x, y, w, h } = this.layout[idx]
+        const layoutXYWH = [x, y, w, h]
+
+        if (xywh.toString() === layoutXYWH.toString()) return
```

Comment:
This wont be ok since ['1','2','3'].toString is equal to ['1','23']. Best to just join with a separator and compare